main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -74,6 +74,8 @@ HistoryPanel.prototype._render = function() {
   this._div.empty().unbind().html(DOM.loadHTML("core", "scripts/project/history-panel.html"));
 
   var elmts = DOM.bind(this._div);
+
+  HistoryPanel.prototype.showHideFacetSplash();
```

Comment:
@Babi-B: now, you are no longer working in the facet panel, but rather the history panel. Your naming of function should show that.